pkgs/applications/misc/styx/default.nix

```diff
@@ -15,27 +13,18 @@ stdenv.mkDerivation rec {
 
   server = "${caddy}/bin/caddy";
   linkcheck = "${linkchecker}/bin/linkchecker";
```

Comment:
Where are those used? I couldn't spot them in them in the install script.

Reply:
server is used [here](https://github.com/styx-static/styx/blob/1d45ff744a34b09467dcd81adcab1737cb5dc357/src/styx.sh#L192)  and `linkcheck` [here](https://github.com/styx-static/styx/blob/1d45ff744a34b09467dcd81adcab1737cb5dc357/src/styx.sh#L190). Both are used in the primary `styx.sh` file.

Reply:
How does upstream envisioned this to work? There is surely some configure script or makefile?

Reply:
Here is the [upstream build](https://github.com/styx-static/styx/blob/789127c93bf6a4a337ff1874c986429ac215f5b6/src/app/cli.nix#L19-L23).